src/odemis/driver/test/xt_client_test.py

```diff
@@ -31,7 +31,7 @@
 from odemis import model
 
 from odemis.driver import xt_client
-from odemis.driver.xt_client import DETECTOR2CHANNELNAME, XTTKDetector
+from odemis.driver.xt_client import XTTKDetector
```

Comment:
It seems as if XTTKDetector remains unused, however I didn't put it there/neither delete anything related to it. So I'll leave it.

Reply:
You can remove it, it was left over from a test that got removed. I first created a separate test for the XTTKDetector, however in the constructor of the SEM class depending on if XTToolkit is used the correct Detector is picked. Therefore the tests regular acquisition tests in TestMicroscope also apply to the XTTKDetector.